src/github/issuesParser.ts

```diff
@@ -2,12 +2,30 @@ import { debug } from "@actions/core";
 import { GitHub } from "@actions/github/lib/utils";
 import moment from "moment";
 
+//** Handles the problem of pagination */
+const getAllIssues = async (octokit: InstanceType<typeof GitHub>, repo: Repo): Promise<IssueData[]> => {
+    let currentPage = 1;
+    const { data } = await octokit.rest.issues.listForRepo({ ...repo, per_page: 100, state: "open", page: currentPage });
+    let issues = data;
+    let fullPage = issues.length > 99;
+    while (fullPage) {
+        currentPage++;
+        debug(`Iterating on page ${currentPage} with ${issues.length} issues`);
+        const { data } = await octokit.rest.issues.listForRepo({ ...repo, per_page: 100, page: currentPage, state: "open" });
```

Comment:
nits: this could be moved to its own method to not repeat the settings (state, per_page, page), and `99` perhaps could go to to a const